docs/src/operate/c8y/software-management.md

```diff
@@ -112,6 +112,19 @@ If you are upgrading %%te%% from any version prior to `1.0.0` (including `1.0.0-
 
 ## Configuration {#configuration}
 
+### Advanced Software Management
```

Comment:
An introduction is missing to tell what makes this feature *advanced*.

Reply:
something like:?

The Advanced Software Management feature in Cumulocity IoT enhances the traditional software management capabilities by providing more flexible and comprehensive control over software operations on devices. This advanced feature supports detailed software types and more granular control over software management processes, allowing for improved management and monitoring of software across diverse device fleets.

Reply:
I'm not sure if we should explain what is advanced software management. It's purely c8y's feature and terminology, and for thin-edge, it's just an endpoint difference. Isn't it enough to give a link to the c8y docs (there's a link in the first sentence of this paragraph)

Reply:
Yes, a link to Cumulocity docs is enough.

Reply:
Updated the description [861c9bd](https://github.com/thin-edge/thin-edge.io/pull/2985/commits/861c9bd37ba506fac7ec6c702e98c1da7b08725f)